stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/Method.java

```diff
@@ -32,20 +49,31 @@
     private String methodName;
     private String methodId;
     private String inputType;
+    private boolean primitiveInput;
     private String outputType;
     private MethodDescriptor.MethodType methodType;
+    private String httpMethod;
+    private String httpPath;
+    private String httpBody;
+    private List<Param> paramSet;
 
-    private Method(String methodName, String methodId, String inputType, String outputType, MethodDescriptor
-            .MethodType methodType) {
+    private Method(String methodName, String methodId, String inputType, boolean primitiveInput, String outputType,
+                   MethodDescriptor.MethodType methodType, String httpMethod, String httpPath, String httpBody,
+                   List<Param> paramSet) {
         this.methodName = methodName;
         this.methodType = methodType;
         this.methodId = methodId;
         this.inputType = inputType;
+        this.primitiveInput = primitiveInput;
         this.outputType = outputType;
+        this.httpMethod = httpMethod;
+        this.httpPath = httpPath;
+        this.httpBody = httpBody;
+        this.paramSet = paramSet;
     }
 
-    public static Method.Builder newBuilder(String methodId) {
-        return new Method.Builder(methodId);
+    public static Method.Builder newBuilder(String methodId, Map<String, Message> messageMap) {
+        return new Method.Builder(methodId, messageMap);
```

Comment:
Instead of passing another parameter to this method. Shall we add a new setter method in `Method.Builder` class like `setMethodDescriptor`.

Reply:
Added a setter method, and removed the additional parameter